src/components/grid-list/grid-list.spec.ts

```diff
@@ -0,0 +1,384 @@
+import {
+    it,
+    describe,
+    expect,
+    beforeEach,
+    inject,
+} from '@angular/core/testing';
+import {TestComponentBuilder, ComponentFixture} from '@angular/compiler/testing';
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+
+import {MD_GRID_LIST_DIRECTIVES, MdGridList} from './grid-list';
+import {MdGridTile} from './grid-tile';
+
+describe('MdGridList', () => {
+  let builder: TestComponentBuilder;
+
+  beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
+    builder = tcb;
+  }));
+
+  it('should throw error if cols is not defined', () => {
+    var template = `<md-grid-list></md-grid-list>`;
+    return builder.overrideTemplate(TestGridList, template)
```

Comment:
Rather than doing `return` on each build, you can simply make each test `async` and they should auto finish because zones.